### PR TITLE
first go at article tag (html)

### PR DESCRIPTION
**Pull request by short-matthew-f**

### CODE REVIEW

File: content/html/glossaries/html-tag-article.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+Title: "<article>"
+Subjects:
+  - "Web Development"
+  - "Web Design"
+Tags:
+  - "Semantic"
+  - "HTML Tags"
+Catalog Content:
+  - "https://www.codecademy.com/learn/learn-html"
```
Comment (sonnynomnom) [inline content/html/glossaries/html-tag-article.md:10]: Add:
- "https://www.codecademy.com/learn/paths/web-development"

File: content/html/glossaries/html-tag-article.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+Title: "<article>"
+Subjects:
+  - "Web Development"
+  - "Web Design"
+Tags:
+  - "Semantic"
+  - "HTML Tags"
```
Comment (sonnynomnom) [inline content/html/glossaries/html-tag-article.md:8]: Can we just do "Tags" here?

File: content/html/glossaries/html-tag-article.md (hunk 1 above)
Comment (sonnynomnom) [inline content/html/glossaries/html-tag-article.md:10]: Add:

"https://www.codecademy.com/learn/paths/web-development"

File: content/html/glossaries/html-tag-article.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+```html
+<article>
+  <!-- Article content goes inside the tag. -->
+</article>
+``` 
+
+`<article>` is a tag that wraps around a block of HTML giving it semantic meaning. It has no special attributes, and by default will act similarly to a `<div>`.
+
+## Examples
```
Comment (sonnynomnom) [inline content/html/glossaries/html-tag-article.md:27]: Example

File: content/html/glossaries/html-tag-article.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+Suppose that there is a need to layout multiple posts about coding practices for a new site. It would be semantically correct to use the `<article>` tag, as below: 
+
+```html
+<h1>FizzBuzz Feed</h1>
```
Comment (sonnynomnom) [inline content/html/glossaries/html-tag-article.md:32]: love it

File: content/html/glossaries/html-tag-article.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+Tags:
+  - "Semantic"
+  - "HTML Tags"
+Catalog Content:
+  - "https://www.codecademy.com/learn/learn-html"
+---
+
+## Definition 
+
+The semantic HTML tag `<article>` represents a part of a page which is self-contained, and could be published elsewhere. Some common uses of the `<article>` tag  include blog posts or magazine articles.
```
Comment (sonnynomnom) [inline content/html/glossaries/html-tag-article.md:15]: The `<article>` semantic tag represents a part of a page that is self-contained and could be published elsewhere. Some common uses include blog posts or magazine articles.

File: documentation/tags.md [hunk 6]
```diff
@@ -115,6 +115,7 @@ Headings
 Heap  
 Heroku  
 Histograms  
+HTML Tags
```
Comment (sonnynomnom) [inline documentation/tags.md:118]: Maybe just Tags?

**Review by sonnynomnom — COMMENTED** [review]
Take a look at this second. I also Slacked you a question!

File: content/html/glossaries/html-tag-article.md (hunk 4 above)
Comment (short-matthew-f) [reply content/html/glossaries/html-tag-article.md:32]: <3

**Pull request merged**